src/pyOpenMS/pxds/AcquisitionInfo.pxd

```diff
@@ -19,8 +26,9 @@ cdef extern from "<OpenMS/METADATA/AcquisitionInfo.h>" namespace "OpenMS":
         String getMethodOfCombination() nogil except +
         void setMethodOfCombination(String method) nogil except +
 
-        Size size() nogil except + #wrap-doc:Number a Acquisition objects
+        Size size() nogil except +
+        #void reserve(size_t n) nogil except + 
         Acquisition operator[](int) nogil except + # wrap-upper-limit:size()
-        void push_back(Acquisition)  nogil except + #wrap-doc:Append a Acquisition object
-        void resize(size_t n) nogil except +
-
+        #void assign(libcpp_vector[Acquisition].iterator, libcpp_vector[Acquisition].iterator) nogil except + # wrap-ignore
```

Comment:
what happened to the removed members? e.g. how to add an Aquisition object?

Reply:
yes, please keep `resize` and `push_back`

Reply:
I needed read-access only (hence the name of the PR), so I had fewer things that can go wrong. At time of construction, there was no such functionality, as is reflected by the version in pypi. 
#4276 wasn't mentioned yet, which does almost what this PR does, except testing and a bit more metainfo access.

Reply:
And it implements push_back and resize etc. But testing it I found that to be not working.